content/millennium/r4/entities/consent.md

```diff
@@ -0,0 +1,191 @@
+
+
+Notes:
+
+- Either the `_id` or one of the following parameters must be provided: `actor`, `identifier`, `patient.identifier`, or `patient`.
+
+- The `actor`, `identifier`, `patient.identifier`, or `patient` parameters may be provided exactly once and may have only a single value.
+
+- The `_include` parameter may be provided once for each resource to be included. Example: `_include=Consent:actor&_include=Consent:patient`
+
+- Additional OAuth2 scopes are required when the `_include` parameter is provided. To include with `Consent:actor` or `Consent:patient` values, the OAuth2 token must include scopes for one or both `RelatedPerson.read` and `Patient.read`, respectively, depending on the combination of resources to be included.
+
+### Headers
+
+ <%= headers %>
+
+### Example retrieve consent
+
+#### Request
+
+    GET https://fhir-open.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/Consent?patient=12742397
+
+#### Response
+
+<%= headers status: 200 %>
+<%= json(:r4_consent_bundle) %>
+
+<%= disclaimer %>
+
+### Search by Id Example
+
+#### Request
+
+    GET https://fhir-open.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/Consent?_id=AR-12724069-12724066
+
+#### Response
+
+<%= headers status: 200 %>
+<%= json(:r4_consent_id_bundle) %>
+
+<%= disclaimer %>
+
+### Search by Actor Example
+
+#### Request
+
+    GET https://fhir-open.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/Consent?actor=12724066-12724069
+
+#### Response
+
+<%= headers status: 200 %>
+<%= json(:r4_consent_actor_bundle) %>
+
+<%= disclaimer %>
+
+### Search by Include Example
+
+#### Request
+
+    GET https://fhir-open.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/Consent?patient=12742397&_include=Consent:patient&_include=Consent:actor
+
+#### Response
+
+<%= headers status: 200 %>
+<%= json(:r4_consent_include_bundle) %>
+
+<%= disclaimer %>
+
+#### Patient Authorization Request
+
+#### Request
+
+    GET https://fhir-open.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/Consent?patient=12742397
+
+#### Response
+
+<%= headers status: 200 %>
+<%= json(:r4_consent_patient_bundle) %>
+
+<%= disclaimer %>
+
+### Errors
+
+The common [errors] and [OperationOutcomes] may be returned.
+
+## Retrieve by id
+
+List an individual Consent by its id:
+
+    GET /Consent/:id
+
+### Authorization Types
+
+<%= authorization_types(provider: true, patient: true, system: true) %>
+
+### Headers
+
+<%= headers %>
+
+### Example
+
+#### Request
+
+    GET https://fhir-open.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/Consent/AR-12724069-12724066
+
+#### Response
+
+<%= headers status: 200 %>
+<%= json(:r4_consent_entry) %>
+
+<%= disclaimer %>
+
+#### Patient Authorization Request
+
+#### Request
+
+    GET https://fhir-open.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/Consent/AR-12724069-12724066
```

Comment:
Update it to https://fhir-ehr.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/Consent/AR-12724069-12724066

Reply:
https://github.com/cerner/fhir.cerner.com/pull/509/commits/4b0037b680117517803a7dcb189943d35ddb8aaf
comment addressed